src/devices/HuskyLens/samples/HuskyLens.Samples.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.IO.Ports;
+
+namespace Iot.Device.HuskyLens.Samples
+{
+    internal class Program
```

Comment:
So as we've been moving to C# 9 and modern way of writing code, you can go for this. In short, remove the name space, remove the class, remove the main, keep the functions. You'll find examples in the repo.